scrapi/base/__init__.py

```diff
@@ -110,26 +136,29 @@ def get_contributors(self, result):
         or creators in the OAI namespaces """
 
         contributors = result.xpath(
-            '//dc:contributor/node()', namespaces=self.NAMESPACES) or ['']
```

Comment:
Right! Good point, removed, because xpath returns just an empty list with no elements found anyhow. Changed to simplify for loop below.